spec/views/curation_concerns/base/show.html.erb_spec.rb

```diff
@@ -16,6 +16,7 @@
       w.artist = ['Boo']
       w.author = ['Cheep Cheep']
       w.addressee = ['Koopa']
+      w.editor = ['Cackletta']
       w.engraver = ['Thwump']
```

Comment:
This is a spec file, as long as you're adding this to the metadata in the file to be shown... add it to the test that makes sure it is on the page too, like it makes sure the others are on the page?

This may apply to other spec files where you added the metadata but didn't necessarily add the test too. 

If it's HARD to add the test... skip it. But in this file it's just following the exact construction used to check other metadata is on page I think?

Reply:
Fixed! Thanks for catching this!